.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
```

Comment:
I think you may need `parallel = True` and `branch = True` as well.

Reply:
`branch = True`, yes I should add that. `parallel = True` is being [set by `pytest-cov`](https://pytest-cov.readthedocs.io/en/latest/config.html#configuration).